packages/consumption/src/modules/attributes/AttributesController.ts

```diff
@@ -1493,10 +1492,21 @@ export class AttributesController extends ConsumptionBaseController {
     }
 
     private isValidTag(tag: string, validTags: Record<string, IAttributeTag> | undefined): boolean {
-        const customTagPrefix = `x${AttributesController.TAG_SEPARATOR}`;
-        if (tag.toLowerCase().startsWith(customTagPrefix)) return true;
+        const customPrefix = "x:";
```

Comment:
I'd prefer to define `private static readonly TAG_SEPARATOR = ":";` (see other comment) and use it here and for the other prefix definitions.

```suggestion
        const customPrefix = `x${AttributesController.TAG_SEPARATOR}`;
```